lib/awful/rules.lua.in

```diff
@@ -193,37 +194,152 @@ function rules.apply(c)
     rules.execute(c, props, callbacks)
 end
 
+--- Get a property value
+-- @param c The client.
+-- @param property The property name
+-- @param value The property value
+-- @param ignore_function A table of properties where functions should not be executed
+local funcion get_property(c,property,value,ignore_function)
```

Comment:
s/funcion/function/ ^^

Also please add whitespace around punctuation (comma in the argument list).